packages/origin-247-certificate/src/onchain-certificate/blockchain-properties.service.ts

```diff
@@ -6,6 +6,11 @@ import { getProviderWithFallback } from '@energyweb/utils-general';
 import { waitForState } from '../utils/wait.utils';
 import { getConfiguration } from './config/configuration';
 
+type DeployParameters = {
```

Comment:
this could be imported from `@energyweb/issuer`

Reply:
implemented on [f4a209b](https://github.com/energywebfoundation/origin-247-sdk/pull/104/commits/f4a209bc63a412fb229c376e64d8a1496a6832c5)